etc/samples/atoms_generic.sample.yaml

```diff
@@ -0,0 +1,26 @@
+# This schema is not used stand-alone, it is a sub-section of "tendrl_definitions_<sds_nvr>.yaml" | "tendrl_definitions_generic.sample.yaml
+
+---
+sds_version: gluster-3.8.3
+object_details:
+  generic_object:
+    atoms:
+        create:
+          enabled: True
+          uuid: statically generated per atom
+          name: "Name"
+          # List of flows this of which this atom can be part of
+          flows: ["TendrlFlow1"]
+          run: "tendrl.node_agent.atoms.cpu.create.Create"
+          type: "get"
+          pre_run : ["tendrl.node_agent.atoms.<atom_name>.<boolean_function>1"...]
+          post_run: ["tendrl.node_agent.atoms.<atom_name>.<boolean_function>1"...]
+          name: Meaningful name for this atom, should be something that is distinguishable and understandable for notification, debugging, storing and any other similar purposes
+          outputs:
+          - "A map of type {object.attr1 : {required: True}, object.attr2:{required: False}}}} that this will be providing (or could provide) to others, used to correlate and associate the thing/s this atom produces, if it produces anything at all."
+
+          # Required input object.attribute list
+          inputs:
```

Comment:
I am bit confused here I feel because volume creation we can pass stripe_count if we want a striped volume, else with only "volname" and "brickdetails" it would create a distributed volume by default.
How to handle this kind of scenario?